src/pages/userPage/Secretaries/UserCitySecretaryTable.tsx

```diff
@@ -1,81 +1,106 @@
-import React, { useEffect, useState, PropsWithRef } from 'react';
-import { Table, Spin, Input } from 'antd';
+import React, {useEffect, useState, PropsWithRef} from 'react';
+import {Table, Spin, Input, Empty} from 'antd';
 import columns from './columnsCIties';
-import { getUsersAdministrations, getusersPreviousAdministrations } from "../../../api/citiesApi";
+import {getUsersAdministrations, getusersPreviousAdministrations} from "../../../api/citiesApi";
+import {getUsersPreviousAdministrations} from "../../../api/regionsApi";
+import Modal from "antd/lib/modal";
 
 
 interface props {
 
-  UserId: string;
+    UserId: string;
 }
 
-export const UserCitySecretaryTable = ({  UserId }: props) => {
-
-
-  const [data, setData] = useState<any>([{
-    id: '',
-    user: '',
-    adminType: '',
-    startDate: '',
-    endDate: '',
-    city: {
-      id: '',
-      name: ''
-    }
-  }]);
-
-
-  const [prevData, setPrevData] = useState<any>([{
-    id: '',
-    user: '',
-    adminType: '',
-    startDate: '',
-    endDate: '',
-    city: {
-      id: '',
-      name: ''
+export const UserCitySecretaryTable = ({UserId}: props) => {
+    const [isLoadingActive, setIsLoadingActive] = useState<boolean>(true);
+    const [isLoadingPrev, setIsLoadingPrev] = useState<boolean>(true);
+
+    const [data, setData] = useState<any>([{
```

Comment:
It worth creating a separate model for this entity, instead of hardcoding it here, there are a lot of them in project already